contracts/README.md

```diff
@@ -19,41 +19,47 @@ reason, use the following commands:
-  cosmwasm/rust-optimizer:0.10.5 ./contracts/burner
+  cosmwasm/rust-optimizer:0.10.7 ./contracts/burner
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_hackatom",target=/code/contracts/hackatom/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.10.5 ./contracts/hackatom
+  cosmwasm/rust-optimizer:0.10.7 ./contracts/hackatom
+
+docker run --rm -v "$(pwd)":/code \
+  --mount type=volume,source="devcontract_cache_ibc_reflect",target=/code/contracts/ibc-reflect/target \
+  --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+  cosmwasm/rust-optimizer:0.10.7 ./contracts/ibc-reflect
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_queue",target=/code/contracts/queue/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.10.5 ./contracts/queue
+  cosmwasm/rust-optimizer:0.10.7 ./contracts/queue
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_reflect",target=/code/contracts/reflect/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.10.5 ./contracts/reflect
+  cosmwasm/rust-optimizer:0.10.7 ./contracts/reflect
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_staking",target=/code/contracts/staking/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.10.5 ./contracts/staking
+  cosmwasm/rust-optimizer:0.10.7 ./contracts/staking
 ```
 
 ## Entry points
 
 The development contracts in this folder contain a variety of different entry
 points in order to demonstrate and test the flexibility we have.
 
-| Contract | Macro                                         | Has `query` | Has `migrate` |
-| -------- | --------------------------------------------- | ----------- | ------------- |
-| burner   | `#[entry_point]`                              | no          | yes           |
-| hackatom | [`create_entry_points_with_migration!`][cepm] | yes         | yes           |
-| queue    | mixed<sup>1</sup>                             | yes         | yes           |
-| reflect  | [`create_entry_points!`][cep]                 | yes         | no            |
-| staking  | `#[entry_point]`                              | yes         | no            |
+| Contract    | Macro                                         | Has `query` | Has `migrate` |
+| ----------- | --------------------------------------------- | ----------- | ------------- |
+| burner      | `#[entry_point]`                              | no          | yes           |
+| hackatom    | [`create_entry_points_with_migration!`][cepm] | yes         | yes           |
+| ibc_reflect | `#[entry_point]`                              | yes         | no            |
```

Comment:
This casing mix is a bit annoying (I know where it comes from though). If we want to keep the dash for the create name (or marketing name), then I'd use it here as well.

```suggestion
| ibc-reflect | `#[entry_point]`                              | yes         | no            |
```